app/controllers/api/v1/cars.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V1::CarsController < ApplicationController
+  before_action :set_cars, only: %i[show destroy]
+  before_action :set_user, only: %i[create destroy]
+
+  # GET /cars/1
+  def show
+    render json: @car
+  end
+
+  private
+
+  # Use callbacks to share common setup or constraints between actions.
+  def set_car
+    @car = car.find(params[:id])
```

Comment:
- Kindly make car here uppercase so it references the model correctly
```suggestion
    car = Car.find(params[:id])
```

- Also, since the frontend is being made on react, there is no need to prepend your variables with `@`. This is only necessary when working with erb files